docs/tutorial.md

```diff
@@ -0,0 +1,198 @@
+# Oria Operator Tutorial
+
+An in-depth walkthrough of building and running a oria-operator with memcached-operator.
+
+## Prerequisites
+
+- [Operator SDK](https://sdk.operatorframework.io/docs/installation/) v1.21.0 or newer
+- User authorized with `cluster-admin` permissions so that you can install an operator, CRDs and change rbac.
+- [GNU Make](https://www.gnu.org/software/make/)
+- Create an [Kind Cluster](https://kind.sigs.k8s.io/)
+- Install [kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+
+## Overview
+
+We will create a sample project to let you know how it works and this sample will:
+
+- Create a `memcached-operator` using below mentioned steps. For more infomation refer to this [tutorial](https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/)
+- Generate the manifests using `config/default`
+- Comment out `ClusterRole` and `ClusterRoleBinding`. 
+- Create a `ClusterRole` template in a `ScopeTemplate` and associate it with a `ScopeInstance`
+- Run the operator, the `memcached-operator` will start complaining about `RBAC`.
+- Run the `oria-operator`, it will create `ClusterRole` and `ClusterRoleBinding`. This will make sure that `memcached-operator` will not complain about `RBAC`. 
```

Comment:
I like this summary. Gives the user a good idea of what they are about to do.

Reply:
cool